packages/address-book-controller/src/AddressBookController.ts

```diff
@@ -126,17 +125,17 @@ export class AddressBookController extends BaseController<
    *
    * @param address - Recipient address to add or update.
    * @param name - Nickname to associate with this address.
+   * @param addressType - Contact's address type.
    * @param chainId - Chain id identifies the current chain.
    * @param memo - User's note about address.
    * @returns Boolean indicating if the address was successfully set.
-   * @param addressType - Contact's address type.
    */
   set(
     address: string,
     name: string,
+    addressType: AddressType,
     chainId = '1',
     memo = '',
-    addressType = AddressType.unknown,
```

Comment:
Could we leave this in last place? We would still want this to be the last parameter to avoid breaking changes.

Reply:
I updated my PR and set addresType optional.
I set the default value to AddressType.externallyOwnedAccounts. Do you think it's a good idea?